artiq/dashboard/experiments.py

```diff
@@ -73,6 +73,8 @@ def __init__(self, manager, dock, expurl):
 
             entry = procdesc_to_entry(argument["desc"])(argument)
             widget_item = QtWidgets.QTreeWidgetItem([name])
+            if argument["tooltip"]:
```

Comment:
The inconsistency is from when I was trying to make the browser able to open old results files without the tooltip entry in expid - I decided this was unnecessary. 

I will correct this inconsistency.